capstone/src/main/java/com/google/step/clients/ScoreClient.java

```diff
@@ -67,10 +67,12 @@ private double getSystemAverage(List<RestaurantPageViews> allPageViews) {
         double count = 0;
 
         for (RestaurantPageViews restaurant : allPageViews) {
+            // Can't be zero because if the RestaurantPageViews exists then it will have at a minimum
```

Comment:
I guess for this I was wondering what would happen if there are not yet any page views in the system. It certainly is an unlikely event, but it is possible. It is true that at this point, the count can no longer be 0, but in the case that there aren't any page views, this loop will be skipped entirely, and the count will still be 0.

Reply:
Yup, there's an initial if statement that checks if the return from getting all the pageViews is null then just return an emtpy list, so this wouldn't even get called.